v5/core/base_service.go

```diff
@@ -433,7 +433,7 @@ func (service *BaseService) Request(req *http.Request, result interface{}) (deta
 		if httpResponse.Body != nil {
 			var readErr error
 
-			defer httpResponse.Body.Close()
+			defer httpResponse.Body.Close() // #nosec G307
```

Comment:
Had to also fix a few gosec-related errors since a new version of gosec is reporting a few new false positives.